Makefile

```diff
@@ -1,6 +1,6 @@
-
+SHELL=/bin/bash
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= platform9/luigi:master
```

Comment:
shouldn't IMG tag pull branch/tag info dynamically ?  Also probably add BUILD_NUMBER ?

Reply:
Yes we spoke, I will make the change